dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -481,12 +502,19 @@ CDefaultContextMenu::AddStaticContextMenusToMenu(
 
         if (info.Verb.CompareNoCase(L"open") == 0)
         {
-            /* override default when open verb is found */
-            fState |= MFS_DEFAULT;
             idResource = IDS_OPEN_VERB;
+            fState |= MFS_DEFAULT; /* override default when open verb is found */
+            forceFirstPos++;
         }
         else if (info.Verb.CompareNoCase(L"explore") == 0)
+        {
             idResource = IDS_EXPLORE_VERB;
+            if (uFlags & CMF_EXPLORE)
```

Comment:
Should this also include a check if there was no default set before?
`explore` will now take precedence over `open`

Reply:
> `explore` will now take precedence over `open`

![XP](https://github.com/reactos/reactos/assets/138629473/3bee274f-802a-40a1-8ff2-d5dca473caf7)